config_examples/tail_sampling.yaml

```diff
@@ -19,14 +19,19 @@ processors:
       - name: keep-slow-traces
         type: latency
         latency: {threshold_ms: 500}
+    decision_wait: 30s
```

Comment:
This is the default value, but I had to add to be able to easily string replace it for the test. This interval is too long and makes the test timeout, as the default `go test` timeout is also 30s. Let me know if you have a better idea for this.

Reply:
Yep agreed. I will leave it like this then.